fvtest/compilertriltest/TypeConversionTest.cpp

```diff
@@ -670,3 +672,274 @@ INSTANTIATE_TEST_CASE_P(TypeConversionTest, Int64ToInt32, ::testing::Combine(
     ::testing::ValuesIn(TRTest::const_values<int64_t>()),
     ::testing::Values(
         std::make_tuple("l2i", l2i) )));
+
+float ibits2f(int32_t x) {
+    union {
+        float f;
+        int32_t i;
+    } fvalue;
+
+    fvalue.i = x;
+    return fvalue.f;
```

Comment:
@Leonardo2718 By using `memcpy`, do you mean code like this?
```
float ibits2f(int32_t x) {
    float f;
    memcpy(&f, &x, sizeof(x));
    return f;
}
```

Reply:
FYI.  fvtest/compilertest/tests/X86OpCodesTest.cpp has similar code using `union`.
```
      union
         {
         float    f;
         uint32_t ui32;
         } gold, result;
      gold.f = sqrt(floatDataArray[i]);
      result.f = _fSqrt(floatDataArray[i]);
      OMR_CT_EXPECT_EQ(_fSqrt, gold.ui32, result.ui32);
```

Reply:
Yes, that's what I meant for `memcpy`.

I'm not terribly worried about the use of a union in the test compiler because the plan is for the Tril tests to replace those tests. :slightly_smiling_face: